studentvue/tests/__init__.py

```diff
@@ -0,0 +1 @@
+from .test_course_history import TestCourseHistory
```

Comment:
When I run `python3 -m unittest`, it's detecting this and running the test the second time. Removing the import fixes this.